lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -594,7 +588,8 @@ def join_app_task_ujs(self, exec_tasks, exec_apps, ujs_jobs):
                     break
 
             # set the run/running/in_queue time
-            if u_j_s['job_state'] == 'completed' or u_j_s['job_state'] == 'suspend':
+            if (u_j_s['job_state'] == 'completed' or
+                    u_j_s['job_state'] == 'suspend'):
```

Comment:
change it to 'suspended'

Reply:
explained right above...it's called 'suspend' in one of the source dbs.